experiments/prebuilt_caches.py

```diff
@@ -0,0 +1,65 @@
+"""
+Script to create tokenized subcaches for speedrun.
+
+This script is used to create subcaches of the fineweb-edu dataset for use in Marin Speedrun.
```

Comment:
If this is going to be this specific, probably this should be in `experiments/speedrun`?